.rubocop.yml

```diff
@@ -1,11 +1,7 @@
 AllCops:
-  Include:
-    - 'Gemfile'
```

Comment:
Is the `Gemfile` automatically included, or do we want it not to be checked for some reason?